test/conformance/ptp/ptp.go

```diff
@@ -884,7 +947,7 @@ func enablePTPEvent() error {
 }
 
 func ptpEventEnabled() bool {
-	ptpConfig, err := client.Client.PtpV1Interface.PtpOperatorConfigs(PtpLinuxDaemonNamespace).Get(context.Background(), "default", metav1.GetOptions{})
+	ptpConfig, err := client.Client.PtpV1Interface.PtpOperatorConfigs(PtpLinuxDaemonNamespace).Get(context.Background(), PtpLinuxDaemonNamespace, metav1.GetOptions{})
```

Comment:
should use ptpConfigOperatorName for name

Reply:
Right! Somehow my changes were ignored by my previous commit.